RevenueCatUI/Purchasing/PurchaseHandler.swift

```diff
@@ -40,6 +93,14 @@ final class PurchaseHandler: ObservableObject {
     @Published
     fileprivate(set) var purchaseResult: PurchaseResultData?
 
+    /// Information used to perform a purchase in the app (rather than in RevenueCat)
```

Comment:
Nit on the comment wording 🙈 

```suggestion
    /// Information used to perform a purchase by the app (rather than by RevenueCat)
```

Reply:
Nit away!!